VideoWeb/VideoWeb/Controllers/ConferencesController.cs

```diff
@@ -53,10 +53,14 @@ public class ConferencesController : Controller
         [ProducesResponseType(typeof(List<ConferenceForJudgeResponse>), (int)HttpStatusCode.OK)]
         [ProducesResponseType((int)HttpStatusCode.BadRequest)]
         [SwaggerOperation(OperationId = "GetConferencesForJudge")]
-        [Authorize(AppRoles.JudgeRole)]
```

Comment:
Please bring this back and use the judicial role instead or both

Reply:
Done, add new authorize policy